extensions/wikia/WikiaLogo/WikiaLogoHelper.class.php

```diff
@@ -47,16 +47,14 @@ public function getCentralUrlForLang( $lang ) {
 	 * @return bool|GlobalTitle
 	 */
 	public function getCentralWikiUrlForLangIfExists( $lang ) {
-		try {
-			return GlobalTitle::newMainPage( $this->wikiCorporateModel->getCorporateWikiIdByLang( $lang ) );
-		} catch ( Exception $ex ) {
-			return false;
-		}
+		$city_id = $this->wikiCorporateModel->getCorporateWikiIdByLang( $lang );
+		return $city_id ? GlobalTitle::newMainPage( $city_id ) : false;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the literal "false" boolean value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1125)